### PR TITLE
[FIX] What is a composite instance? Change to measurement for non MRI modalities?

### PR DESCRIPTION
**Pull request by rob-luke**
What is a composite instance? I had never heard this term before and don't believe it is used in EEG or fNIRS. I googled it and it appears to be a DICOM specific term and thus fMRI specific? But I found the terminology in EEG MEG iEEG and Physiological pages.

I suggest either changing it to measurement (as in this PR), or adding a definition somewhere in the specification.

This is an issue as a PR. Hope that's ok.

### CODE REVIEW

File: src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md
Comment (sappelhoff) [inline src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md:141]: I think I would keep the text for the different fields (here `DeviceSerialNumber`) the same everywhere. "measurements" would work here as well.

**Review by sappelhoff — COMMENTED** [review]
thanks @rob-luke, this looks sensible.

Note that https://github.com/bids-standard/bids-specification/pull/774 aims to render all these tables from the schema, hence my comment to try and make the texts consistent. cc @tsalo is there something from the schema side to do here?

**Comment by sappelhoff** [comment]
PS: linkchecker fails due to biosemi.com failing to renew their certificates (?) ... https fails, but http works. You could just change the link from https to http for now. I am not sure when they'll manage to get their https back up.

File: src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md
Comment (rob-luke) [reply src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md:141]: Here, I just tried to make the minimal change, and the lines above already had `physiological data`. Do you want me to make it measurements in all modality files? Including fMRI?

**Comment by rob-luke** [comment]
FYI @mshader (https://github.com/bids-standard/bids-specification/pull/802#discussion_r642800061)

File: src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md
Comment (sappelhoff) [reply src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md:141]: That's something to discuss together with @tsalo: If we want to render all these metadata fields from the schema, the schema entry should/must be consistent.

Comment (tsalo) [reply src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md:141]: Yes, thank you. The "canonical" metadata definitions in the schema (see #774) are going to just use the generic definition (i.e., "measurement" instead of "physiological data"), so it would be preferable to just stick with "measurement" in the current tables.

Comment (rob-luke) [reply src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md]: Thanks @tsalo

**Comment by rob-luke** [comment]
@sappelhoff I think I addressed all issues. How/who do I ping for review? There seems to be ways to ping a _team_, is that what I should do? And which team?

**Review by tsalo — APPROVED** [review]
The changes LGTM.

**Comment by sappelhoff** [comment]
@rob-luke these are the teams you can ping: https://github.com/orgs/bids-standard/teams/everyone --> all nested under "everyone", so if you ping [at]bids-standard/everyone, EVERYBODY will get a notification ... if you go down the nesting, the amount of pings reduces accordingly.

Usually it's fine to ping one of the maintainers that you last were in touch with about an issue/pr.

**Review by sappelhoff — APPROVED** [review]

**Comment by rob-luke** [comment]
Thanks @sappelhoff for explaining. Looks like this is good to go then :)

**Comment by sappelhoff** [comment]
Yes, I think so too, will merge! Thanks a lot :-)

Just a note for you: Usually we wait about 5 days after the most recent commit before merging a PR, to give the community enough time to see it, see: https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md#rules

But in this case it's a relatively uncontroversial fix that has been approved and open for many days now without many changes.

**Comment by rob-luke** [comment]
Will do @sappelhoff, and Ill be sure to read through that document too.

**Pull request merged**
